### PR TITLE
add print fun

### PR DESCRIPTION
**Pull request by Abdelmagidawad**

### CODE REVIEW

File: Implemented Funs Using JS/Length String.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/ Implemented Function Will Calculate length of String
+// s : The string
+
+function length(s) {
+  let count = 0;
+  while (count < s.length) count++;
```
Comment (MahmoudLkashef) [inline Implemented Funs Using JS/Length String.js]: s.length why you are using s.length your function should do that

File: Implemented Funs Using JS/Length String.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/ Implemented Function Will Calculate length of String
+// s : The string
+
+function length(s) {
```
Comment (MahmoudLkashef) [inline Implemented Funs Using JS/Length String.js]: can you make name of this function more readable 
what's your opinion about getStringLength()

File: Implemented Funs Using JS/String Repeat.js [hunk 3]
```diff
@@ -2,6 +2,11 @@
 // s : The string that will be repeated
 // num :The number of repeating
 
+//
+function print() {
+  console.log("Hello");
+}
+print();
 function stringRepeat(s, num) {
```
Comment (MahmoudLkashef) [inline Implemented Funs Using JS/String Repeat.js]: change param name 
s to value 
num to repeatNum

**Review by MahmoudLkashef — COMMENTED** [review]
please check the comments

File: Implemented Funs Using JS/Length String.js (hunk 1 above)
Comment (Abdelmagidawad) [reply Implemented Funs Using JS/Length String.js]: replaced this condition while without use length method

File: Implemented Funs Using JS/Length String.js (hunk 2 above)
Comment (Abdelmagidawad) [reply Implemented Funs Using JS/Length String.js]: ok

**Review by MahmoudLkashef — APPROVED** [review]

**Pull request merged**